h/static/styles/partials-v2/_search.scss

```diff
@@ -4,12 +4,14 @@
 // of activity queries.
 
 .search-result-container {
+  display: flex;
```

Comment:
This introduces a problem on small screens (small enough that the document buckets must shrink to smaller than their default size), the container ends up being wider than the screen and you get a horizontal scrollbar.

Fortunately setting `flex-direction: column;` on small screens, which I think I'm going to do in the follow-up PR that adds the responsive layout for this, fixes it. So I'll fix this in the follow-up PR.